test/framework/src/main/java/org/elasticsearch/repositories/ESBlobStoreContainerTestCase.java

```diff
@@ -129,16 +129,17 @@ public void testDeleteBlob() throws IOException {
     }
 
     @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/15579")
```

Comment:
Based on discussion with @imotov, this will now be part of a separate follow up PR